geode-web-management/src/main/java/org/apache/geode/management/internal/rest/controllers/ManagementControllerAdvice.java

```diff
@@ -33,9 +45,47 @@
 import org.apache.geode.security.NotAuthorizedException;
 
 @ControllerAdvice
-public class ManagementControllerAdvice {
+public class ManagementControllerAdvice implements ResponseBodyAdvice<Object> {
   private static final Logger logger = LogService.getLogger();
 
+  @Autowired
+  private Jackson2ObjectMapperFactoryBean objectMapperFactory;
+
+  @Override
+  public boolean supports(MethodParameter returnType, Class converterType) {
+    return AbstractJackson2HttpMessageConverter.class.isAssignableFrom(converterType);
```

Comment:
I don't understand why we only return true from supports if the converterType could be assigned to AbstractJackson2HttpMessageConverter. Is it worth a comment?

Reply:
I believe this is because the class information is only added by the "AbstractJackson2HttpMessageConverter".

Reply:
added a comment